install_frontend_dependencies.sh

```diff
@@ -121,9 +121,9 @@ debian_install() {
 }
 
 setup_yarn() {
-    if [ ! -f ".yarnrc.yml" ]; then
-        log "Setting up yarn (yarn set version berry)"
-        yarn set version berry && \
+    if ! yarn -v | grep '^2\.' > /dev/null; then
```

Comment:
looking at the related issue, you are talking about yarn version 1.22, it says here yarn > 2.0.
Is it ok ?
Update diff or issue accordingly

Reply:
It's okay. We need to support any version of yarn (1 or 2) when installing the dependencies, including setting up yarn 2. After that, we only want to support Yarn 2 after installation.